http2curl_test.go

```diff
@@ -118,3 +120,18 @@ func ExampleGetCurlCommand_other() {
 	fmt.Println(command)
 	// Output: curl -X 'PUT' -d '{"hello":"world","answer":42}' -H 'Content-Type: application/json' -H 'X-Auth-Token: private-token' 'http://www.example.com/abc/def.ghi?jlk=mno&pqr=stu'
 }
+
+// Benchmark test for GetCurlCommand
+func BenchmarkGetCurlCommand(b *testing.B) {
+	form := url.Values{}
+
+	for i := 0; i <= 50; i++ {
```

Comment:
I don't know how benchmarks work very well, but I think that you should use this instead ->

```suggestion
	for i := 0; i <= b.N; i++ {
```